Godel.Microservice.CurrencyRates/Business/Dto/CurrencyDto.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Godel.Microservice.CurrencyRates.Business.Dto
+{
+    public class CurrencyDto
+    {
+        public int Id { get; set; }
+
+        public string CurrencyName { get; set; }
```

Comment:
please rename without Currency as class has this already

Reply:
was renamed